client/src/app/core/repositories/agenda/agenda-item-repository.service.ts

```diff
@@ -16,6 +16,8 @@ export interface AgendaListTitle {
     subtitle?: string;
 }
 
+export type AgendaItemType = 1 | 2 | 3;
```

Comment:
Please move it to the `agenda-item.ts` and maybe use it in `ItemVisibilityChoices` as well as change the type of `number` in `AgendaItem`